src/intrinsics/ecma262/NumberPrototype.js

```diff
@@ -144,7 +144,7 @@ export default function(realm: Realm, obj: ObjectValue): void {
     return new StringValue(realm, s + x.toPrecision(p));
   });
 
-  const tsTemplateSrc = "(A).toString()";
+  const tsTemplateSrc = "('' + A)";
```

Comment:
Annoying to have this so many times in our code base. Time to consolidate?

Reply:
There's only three cases of them and they're backed by the Flow enum. Extracting it out into a single place is a good idea though, but it also makes all the other template strings inconsistent – they should all then be imported from a central location to be consistent. Something for a follow up I guess.